### PR TITLE
[Enhancement] [Pipeline] Print profile information when fragment finish

### PR DESCRIPTION
**Pull request by caneGuy**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [x] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by caneGuy** [comment]
cc @stdpain @kangkaisen PTAL thanks

File: be/src/exec/pipeline/pipeline_driver.cpp [hunk 1]
```diff
@@ -332,6 +332,13 @@ void PipelineDriver::finalize(RuntimeState* runtime_state, DriverState state) {
     // last finished driver notify FE the fragment's completion again and
     // unregister the FragmentContext.
     if (_fragment_ctx->count_down_drivers()) {
+        if (FLAGS_minloglevel == 0 /*INFO*/) {
+            std::stringstream ss;
+            // Print profile for this fragment
+            _fragment_ctx->runtime_state()->runtime_profile()->compute_time_in_profile();
+            _fragment_ctx->runtime_state()->runtime_profile()->pretty_print(&ss);
+            LOG(INFO) << ss.str();
```
Comment (kangkaisen) [inline be/src/exec/pipeline/pipeline_driver.cpp:340]: Would better use debug level.  info will print a lot of logs

Comment (kangkaisen) [reply be/src/exec/pipeline/pipeline_driver.cpp:340]: or use a mutable config.

**Review by kangkaisen — APPROVED** [review]

**Review by stdpain — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by caneGuy** [comment]
run starrocks_admit_test

**Comment by caneGuy** [comment]
run starrocks_admit_test

**Comment by kangkaisen** [comment]
run starrocks_admit_test

**Comment by caneGuy** [comment]
run starrocks_admit_test

**Pull request merged**